test/Models/resnet50-bottleneck-block.mlir

```diff
@@ -4,6 +4,10 @@
 // RUN: tpp-run %s -n 10 \
 // RUN:         -print -e resnet50_bottleneck_block -entry-point-result=void | \
 // RUN: FileCheck %s -check-prefix=EXEC
+// XFAIL:*
```

Comment:
Can we just comment the execution run lines instead? So we still run the IR check and make sure it doesn't change?

Making a TODO linking to the issue is still a good idea!

Reply:
Good point, we XFAIL is too broad.